static/src/javascripts/projects/common/modules/navigation/sticky.js

```diff
@@ -228,9 +228,13 @@ define([
 
         this.setScrollDirection(scrollY);
 
+        var $appBanner = $('.site-message--ios, .site-message--android');
+        var appBannerHeight = $appBanner.height();
```

Comment:
It's nice that you use your own bonzo's height method but we have lots of `dim().height` in this file. I am afraid that now you will have to refactor it and use the `.height()` method.